replay_tool/tasks.py

```diff
@@ -146,14 +146,14 @@ def get_local_aoi_extract():
         password={db_password} validateSchemaVersion=no --write-xml file={path}
     '''
     # Write command to named pipe
-    with open('osmosis_command_reader.fifo', 'w') as f:
+    with open('/tmp/osmosis_command/command_reader.fifo', 'w') as f:
         f.write(command)
 
     # wait
     time.sleep(OSMOSIS_COMMAND_TIMEOUT_SECS)
 
     # Read result
-    with open('osmosis_result_reader.fifo') as f:
+    with open('/tmp/osmosis_command/result_reader.fifo') as f:
```

Comment:
the previous `osmosis_command_reader.fifo` was meant to be accessed from host[used by `run.sh`] and also from inside the replay-tool container. Will this change make it available for `run.sh`?

Reply:
Previously we were using runserver and using host code to run the replay-tool-server in **posm-build**. I am now using image code instead which breaks the previous method. Just exposing the `/tmp/osmosis_command/` makes it enough for host/external service to reach the files. So exposing the volume `/tmp/osmosis_command/` should be enough in docker-compose.yml. Let me know what you think.

Reply:
yes, exposing volume will be enough. It's just that I did not see it in this PR.

Reply:
Updated docker-compose.yml https://github.com/posm/posm-replay-server/pull/23/commits/e3632c11cd082b94f8ff6c4be0e7b83999baacde